### DISCUSSION TITLE
Feature request: cache upscaler result

### DISCUSSION THREAD

**Discussion by crazygonuts** [Coyote-A/ultimate-upscale-for-automatic1111#77] [Q&A] [answered]
If there is already a way to do this, I apologize but I couldn't figure it out.

Sometimes with very large images, the upscaler can take a significant amount of time, some upscalers are not fast - swinIR and most especially LDSR - and I have to run the ultimate upscale multiple times with different denoise values to get the best final output.  It would save a lot of time if I could just run the upscaler once and then do the tile generation multiple times with different denoise values.  Is there already a way to do this?

**Comment by Jopezzia** [comment, marked as answer]
You can upscale 1 time on extra with upscaler and send it to img2img. Just use target size - Scale from image size - 1
